.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.12.3-otp-24
-erlang 24.0
+elixir 1.14.5-otp-26
+erlang 26.0
```

Comment:
🥺🥺🥺🥺 please 🥺🥺🥺🥺

Reply:
Adding this to the CI and using it locally is ok but we should also test against earlier Elixir versions as far back as 1.9